nubis/files/nubis-ci-deploy

```diff
@@ -1,123 +1,209 @@
 #!/bin/bash -l
+# shellcheck disable=SC1117
```

Comment:
Does this need to be at the top ?

Reply:
Yes and no. It can be put at the top to apply to the entire file or above each instance it is needed. In this case, every variable that I am pulling in from the environment throws this error. As this is a large number I decided to place it at the top. It could be placed ahead of every one of the variables on line 8 through 43, but that feels less readable to me.